R/randomForest_tidiers.R

```diff
@@ -697,13 +697,25 @@ rf_evaluation <- function(data, ...) {
   tidy(data, model, type = "evaluation", ...)
 }
 
+rf_partial_dependence <- function(df, ...) {
```

Comment:
add testcase for this, or at least add TODO to add test later if you have no time for now.